### ISSUE TITLE
Suggestion: APNG support?

### ISSUE THREAD

**Issue by sbrl** [colinkeenan/silentcast#56]
Since [APNG](https://en.wikipedia.org/wiki/APNG) is now [supported by all major browsers](https://caniuse.com/#feat=apng) and it's [generally smaller than an animated GIF](http://littlesvr.ca/apng/gif_apng_webp.html) (not to mention the lack of a colour limit), it would be great if this tool supported saving to it.

**Comment by colinkeenan** [comment]
I don't have time until maybe a couple of months from now. Can you check if this format is supported by ffmpeg?

**Comment by sbrl** [comment]
Yeah, sure! I did some research a few weeks ago and `ffmpeg` natively supports APNG.

_(Also, [I created a tldr-page for `silentcast`](https://github.com/tldr-pages/tldr/pull/2936) :D)_
